### ISSUE TITLE
Run background apps at startup

### ISSUE THREAD

**Issue by Zac0511** [HeyPuter/puter#428]
The user could open settings, and set some apps that are made to run in background to run when Puter starts. For example, if its an app that is supposed to alert the user that something happened somewhere (like on his NAS if he has one and has an app that can connect to it), those alerts would be working just after Puter has started. The programs could still get stopped from the task manager if needed, or removed from startup from the settings.

**Comment by AtkinsSJ** [comment]
Good idea! It doesn't even have to just be background apps. On Linux I have a notes app that launches on startup too.

**Comment by Zac0511** [comment]
> Good idea! It doesn't even have to just be background apps. On Linux I have a notes app that launches on startup too.

Yeah, it could be for not-background apps too now that i think about that !

**Comment by adamq-q** [comment]
Would be a great feature to have, and happy to contribute.

As a starting point, does anyone know of a proper way to get an app to launch on startup? Even if there's no settings menu  exposed to the user to enable/disable a startup app yet.

**Comment by Zac0511** [comment]
> Would be a great feature to have, and happy to contribute.
> 
> As a starting point, does anyone know of a proper way to get an app to launch on startup? Even if there's no settings menu exposed to the user to enable/disable a startup app yet.

Well the only way i know as of now is with the URL (https://puter.com/[app name])
